pkgs/_analyzer_macros/lib/macro_implementation.dart

```diff
@@ -9,7 +9,7 @@ import 'package:analyzer/src/summary2/macro_declarations.dart' as analyzer;
 import 'package:analyzer/src/summary2/macro_injected_impl.dart' as injected;
 import 'package:dart_model/dart_model.dart';
 import 'package:macro_service/macro_service.dart';
-import 'package:macros/macros.dart';
+import 'package:macros/macros.dart' hide Code;
```

Comment:
If we have two `Code` classes we should really rename one.

Or maybe, it would help to always prefix the one that is used. Cause it is very confusing :)